Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml

```diff
@@ -561,3 +561,13 @@
   - type: Item
     heldPrefix: gold
     
+
+- type: entity
+  parent: IDCardStandard
```

Comment:
same thing

Reply:
will do

Reply:
The cluwne now uses its own id card sprite so it doesnt make sense to inherit from the clownid.